test/lit.cfg

```
@@ -919,8 +919,8 @@ if run_os in ('maccatalyst',):
   target_os_abi = 'macosx'
   target_os_is_maccatalyst = "TRUE"
   config.available_features.add("OS=ios")
-# macOS on ASi uses the stable ABI
-if run_os in ('macosx',) and run_cpu in ('arm64',):
+# macOS and simulators on ASi use the stable ABI
+if (run_os in ('macosx',) or run_environment in ('simulator',)) and run_cpu in ('arm64',):
```

```suggestion
if (run_os in ('macosx',) or 'simulator' in run_environment) and run_cpu in ('arm64',):
```

The simulator check needs to be reworked for `swift_only_stable_abi` to be passed appropriately.

---

Fixed in a9501e08e6c625c6983ad42403ba4fd9e3146633